config.py

```diff
@@ -1,11 +1,16 @@
 import os
 import pymysql
 
+S3_BUCKET                       = "coms4156-strategies"
+S3_LOCATION                     = 'http://{}.s3.amazonaws.com/'.format(S3_BUCKET)
```

Comment:
I love this. It shows efforts to hide instance host name and protects its ip address. well done.